clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -8,10 +8,9 @@ describe("System management page", () => {
     );
   });
 
-  // TODO: Update Cypress test to reflect the nav bar 2.0
-  it.skip("Can navigate to the system management page", () => {
-    cy.visit("/");
-    cy.getByTestId("nav-link-Systems").click();
+  it("Can navigate to the system management page", () => {
+    cy.visit("/system");
```

Comment:
this kind of bypasses what this test was originally for, to be able to navigate _to_ the page via the UI (the other tests do a `cy.visit("/system")` already so have that use case covered)

I think this test should ideally go through the nav bar to get to system